corona-warn-app/cwa-app-ios
# brought hourly fetching back

## Checklist

* [x] This PR does not change text that hasn't been signed off with the RKI. Have a look at the pinned issue [440](https://github.com/corona-warn-app/cwa-app-ios/issues)
* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] If this PR comes from a fork, please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
* [x] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes # 41)
* [x] [Link your Pull Request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) - Pull Requests that are not linked to an issue with you as an assignee will be closed.
* [x] Create Work In Progress [WIP] pull requests only if you need clarification or an explicit review before you can continue your work item.
* [x] Make sure that your PR is not introducing _unnecessary_ reformatting (e.g., introduced by on-save hooks in your IDE)


## Description
This PR brings back hourly fetching. At the moment this only works for developer builds due to technical challenges that have to be addressed first.

**If you want to test this PR:**

- simply set your date to 3 days in the past. 
- Remove the app. 
- Install the app. 
- Launch the app 
- go though onboarding.
- Then enable hourly fetching mode (developer menu). 
- Then set the time to today. 
- Go back to the app and make sure the hourly packages are fetched.
